data_pipeline_api/registry/download.py

```diff
@@ -117,6 +120,8 @@ def download_from_config_file(config_filename: Union[Path, str], token: str) ->
     with open(config_filename, "r") as cf:
         config = yaml.safe_load(cf)
     run_metadata = config.get("run_metadata", {})
+    # Copy the top level data_directory into the run_metadata.
+    run_metadata[RunMetadata.data_directory] = config["data_directory"]
```

Comment:
The download script now emulates the behaviour of the file API in copying the data directory over.